src/main/osd/osd.h

```diff
@@ -190,6 +190,16 @@ typedef enum {
     OSD_GPS_LAP_TIME_PREVIOUS,
     OSD_GPS_LAP_TIME_BEST3,
     OSD_DEBUG2,
+    OSD_CUSTOM_MSG0,
```

Comment:
As above, just use one MSP code and pass an argument for the message instance.

Reply:
position, visibility and other attributes are associated with is

Reply:
See https://github.com/betaflight/betaflight/blob/73b26168cdd7fec45d35e23f10abe3985525314b/src/main/msp/msp.c#L2316 for how to do this.

Reply:
Sorry, I don't understand your comment. This is not related to MSP, this is OSD_ITEM Id. It is used to handle custom text as other OSD items.

Reply:
I was wondering if we could do something similar but there’s no way to pass that parameter so I think you’re right; not relevant.